scripts/index-services.py

```diff
@@ -2,18 +2,21 @@
 """Read services from the API endpoint and write to search-api for indexing.
 
 Usage:
-    index-services.py <stage> --frameworks=<frameworks> [options]
+    index-services.py <stage> --index=<index> --frameworks=<frameworks> [options]
 
-    --serial                                      Do not run in parallel (useful for debugging)
-    --index=<index>                               Search API index name [default: g-cloud-9]
+    <stage>                                       One of dev, preview, staging or production
+    --index=<index>                               Search API index name
```

Comment:
Maybe say that it should be of the format `<framework>-YYYY-MM-DD`.

Reply:
Does it actually need to be though? For local, would it matter?

Reply:
It doesn't matter specifically.

Reply:
Cool. Have updated.